src/main/common/filter.c

```diff
@@ -67,6 +67,32 @@ float pt1FilterApply4(pt1Filter_t *filter, float input, uint8_t f_cut, float dT)
     return filter->state;
 }
 
+// Slew filter with limit
+
+void slewFilterInit(slewFilter_t *filter, float slewLimit, float threshold)
+{
+    filter->state = 0.0f;
+    filter->slewLimit = slewLimit;
+    filter->threshold = threshold;
+}
+
+float slewFilterApply(slewFilter_t *filter, float input)
```

Comment:
I would rather made it dT-dependent, otherwise different sample time will require choosing different parameters of the filter.

Reply:
The parameters can easily be scaled by dT in the init function.

Reply:
If you intend to use it in main PID loop - you'll need to scale them each iteration. Looptime is super unstable in 4K and above on most boards. Especially with OSD or baro on I2C bus.

Reply:
The PID calculation itself and the other filters assume constant `dT`. That's probably something that should be addressed as well.

The slew filter itself is intended to address large gyro changes, especially sign reversals, after a crash, so is less `dT` dependent than the other filters or the PID loop itself.